pkg/compass/compass.go

```diff
@@ -22,6 +22,7 @@ const (
 
 	habitatPathEnv         = "HABITAT_PATH"
 	habitatIdentityPathEnv = "HABITAT_IDENTITY_PATH"
+	defaultHabitatPathUnix = ".habitat"
```

Comment:
I think we should have a function for this already

Reply:
fixed